### PR TITLE
Milestone: Create animals table

### PR DESCRIPTION
**Pull request by Kashif-Saleem-Ghuman**
In this milestone 💪 

- [ ] Created an animals table in SQL.
- [ ] Inserted and queried data in SQL.
- [ ] Screenshots:

thubusercontent.com/111677605/220328724-fbf296f5-2bd7-41f8-a75b-4897d21a63ec.JPG)

943.JPG)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,17 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE IF NOT EXISTS public.animals
+(
+    id integer,
```
Comment (1GORDON) [inline schema.sql]: Great job @Kashif-Saleem-Ghuman 🤝

- [ ] [Optional] You may set the id column not null and autoincremented, and mark it as a primary key.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Kashif-Saleem-Ghuman ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] As stated in the project requirements, kindly add screenshots of your queries' results to the pull request description.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kashif-Saleem-Ghuman** [comment]
HI @1GORDON , thank you for reviewing my PR and providing valuable feedback. 👍

**Review by t-yanick — APPROVED** [review]
Hi @Kashif-Saleem-Ghuman , 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Repo generated using correct template
:heavy_check_mark: Screenshots Google drive link provided

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kashif-Saleem-Ghuman** [comment]
@t-yanick  Thank you for reviewing and approving my PR.

**Comment by t-yanick** [comment]
@Kashif-Saleem-Ghuman please kindly put the images (screenshots) in the PR description before merging. I was not supposed to approve this milestone. Kindly do this please.
You are supposed to put screenshots in the PR description and not a google drive link to where the images have been stored. 

Thanks for understnading

**Comment by Kashif-Saleem-Ghuman** [comment]
@t-yanick I have already merged it, but I will still add the screenshots directly in the PR

**Pull request merged**